crates/sui-indexer/src/apis/governance_api.rs

```diff
@@ -23,53 +22,15 @@ use sui_types::{
     sui_serde::BigInt,
     sui_system_state::{sui_system_state_summary::SuiSystemStateSummary, PoolTokenExchangeRate},
 };
-use tokio::sync::Mutex;
 
 #[derive(Clone)]
 pub struct GovernanceReadApi<T: R2D2Connection + 'static> {
     inner: IndexerReader<T>,
-    exchange_rate_cache: Arc<Mutex<SizedCache<EpochId, Vec<ValidatorExchangeRates>>>>,
 }
 
 impl<T: R2D2Connection + 'static> GovernanceReadApi<T> {
     pub fn new(inner: IndexerReader<T>) -> Self {
-        Self {
-            inner,
-            exchange_rate_cache: Arc::new(Mutex::new(SizedCache::with_size(1))),
-        }
-    }
-
```

Comment:
Also wondering why this file needs to be changed. Is there a bug in this implementation? If not, maybe it's better to leave it alone or improve it in a separate PR?

Reply:
This was a change done by me a while ago while trying to fix this, before I understood it cannot be fixed that way. I can put up a separate PR, not a problem.